qa/L0_server_logs/test.sh

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Do we need a whole new test for this? Can't we just add `--triton-output-path` to L0_analyze?

Reply:
Yes I suppose we could add it to L0_server_launch_modes.